packages/expo-local-authentication/src/LocalAuthentication.types.ts

```diff
@@ -2,7 +2,14 @@ import { Platform } from 'expo-modules-core';
 
 export type LocalAuthenticationResult =
   | { success: true }
-  | { success: false; error: string; warning?: string };
+  | {
+      success: false;
+      /**
+       * One of the following values: `not_enrolled` `user_cancel` `app_cancel` `not_available` `lockout` `no_space` `timeout` `unable_to_process` `unknown` `system_cancel` `user_fallback` `invalid_context` `passcode_not_set` `authentication_failed`
```

Comment:
```suggestion
       * One of the following values: `not_enrolled`, `user_cancel`, `app_cancel`, `not_available`, `lockout`, `no_space`, `timeout`, `unable_to_process`, `unknown`, `system_cancel`, `user_fallback`, `invalid_context`, `passcode_not_set`, `authentication_failed`.
```

I'd suggest formatting correctly.